docs/topics/items.rst

```diff
@@ -240,6 +240,10 @@ Item objects
     Items replicate the standard `dict API`_, including its constructor. The
     only additional attribute provided by Items is:
 
+    .. automethod:: copy
```

Comment:
dict method is present in Python dict. Previous text doesn't match the following docs now - "The only additional attribute provided by Items is", and then instead of this single additional attribute two methods are described.